Sources/Controllers/MessagesViewController.swift

```diff
@@ -82,25 +81,24 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
         setupDelegates()
         addMenuControllerObservers()
         addObservers()
+        addKeyboardObservers()
     }
-    
+
     open override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         isMessagesControllerBeingDismissed = false
+        messageCollectionViewBottomInset = keyboardOffsetFrame.height - safeAreaBottomInset
+        messagesCollectionView.scrollToBottom(animated: true)
     }
-    
+
     open override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         isMessagesControllerBeingDismissed = true
+        messageCollectionViewBottomInset = keyboardOffsetFrame.height - safeAreaBottomInset
+        messagesCollectionView.scrollToBottom(animated: true)
     }
-    
+
     open override func viewDidLayoutSubviews() {
-        // Hack to prevent animation of the contentInset after viewDidAppear
```

Comment:
So, moved the initial bottomInset setting to viewDidAppear lol

Reply:
This is too late to call it. It won’t look how I want it. I’d appreciate it if you keep this PR in scope. It makes it hard to approve because now I can reject it for keyboard related things